.github/drafts/16506/16506_category_slider_outline.md

```diff
@@ -0,0 +1,62 @@
+*** Category Slider ***
```

Comment:
Could you please restructure the document? I think:
Slider widget - a landing page
  - subpages for:
       - category slider
       - number slider
       - range slider
 P.S. - Please evaluate if events and styles are common or different for each slider. Based on that, it can be added to the slider landing page or individual sliders.

Reply:
Sure! I'll give this a try and see how it looks. However, I'm a bit concerned that the sliders are similar enough that the common page will have like, 85% of the docs for each slider and then 15% will be on the slider-specific page and the content will be separated unnecessarily. Will update when this structure change is complete

Reply:
The slider landing page will briefly introduce the slider widget, its usage, and its type. You can then add individual page links for each slider and talk in detail in their respective subpages. I would suggest starting a CR on GitBook and creating the structure. It'll give you a fair idea of how things could be structured. Could you please share the GitBook link in the outline description at the start so that it's easy for @dilippitchika and @yaldram to visualize how the document would be strucutured.